esp/esp/program/modules/handlers/jsondatamodule.py

```diff
@@ -735,7 +737,9 @@ def student_max_count(clslist):
         for g in grades:
             grade_classes = classes.filter(status__gte=0, grade_min__lte=g, grade_max__gte=g)
             grade_sections = prog.sections().filter(status__gte=0, parent_class__in=grade_classes)
-            grade_students = filter(lambda x: x.getGrade(prog)==g, students['enrolled'])
+            grade_students = filter(
+                lambda x: x.getGrade(prog, assume_student=True)==g,
```

Comment:
You'll need to rebase on Anthony's changes again. Looks like there is no longer an ``assume_student`` argument.